scripts/generate_psa_key_attributes.py

```diff
@@ -111,22 +133,73 @@ def pack(self):
             0,  # Reserved, only used if key id encodes owner id
         )
 
+
 def is_valid_hexa_code(string):
     try:
         int(string, 16)
         return True
     except ValueError:
         return False
 
-def main() -> None:
+
+def generate_attr_file(
+    attributes: PlatformKeyAttributes,
+    key_value: str,
+    key_file: str,
+    trng_key: bool,
+    key_from_file: str,
+    bin_out: bool,
+) -> None:
+    metadata_str = binascii.hexlify(attributes.pack()).decode("utf-8").upper()
+
+    if key_file and Path(key_file).is_file():
+        with open(key_file, encoding="utf-8") as file:
+            json_data = json.load(file)
+    else:
+        json_data = json.loads('{ "version": 0, "keyslots": [ ]}')
+
+    if trng_key:
```

Comment:
While we are changing this we should make these mutual exclusive through argparse, not a blocker for this PR though.